v1/api/swagger/components/schemas/HostState.yaml

```diff
@@ -1,5 +1,6 @@
 type: string
 enum:
+  - ''
```

Comment:
? Empty Host state is valid?!

Reply:
this is HostStateNil (like HostSubstateNil).  api this state (along with workflow) to drive hostsm entry.